helpers/announce.js

```diff
@@ -0,0 +1,41 @@
+import { offscreenStyles } from '../components/offscreen/offscreen-styles.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+
+class AnnounceContainer extends LitElement {
+
+	static get styles() {
+		return [ offscreenStyles ];
+	}
+
+	constructor() {
+		super();
+		this._messages = [];
+	}
+
+	announce(text) {
+		this._messages.push(text);
+		this.requestUpdate();
+	}
+
+	render() {
+		return html`<div aria-live="polite" class="d2l-offscreen">
+			${this._messages.map(message => html`<div>${message}</div>`)}
```

Comment:
The messages are rendered in the shadow-DOM of this component.  If they are placed in a slot instead, the entire content of the slot is announced with every addition, which is not what we want.